src/lib.rs

```diff
@@ -63,10 +65,15 @@
 //!         true // keep going to the next frame
 //!     });
 //! }
+//! # }
 //! ```
 
 #![doc(html_root_url = "https://docs.rs/backtrace")]
 #![deny(missing_docs)]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(not(feature = "std"))]
+extern crate core as std;
```

Comment:
FWIW my preferred way at least to add `no_std` support is to not do this, but rather unconditionally use `#![no_std]` and then conditionally `extern crate std` whenever `std` is enabled. Then, when necessary, the `std::prelude::v1::*` module is imported